doc/source/whatsnew/v0.24.0.txt

```diff
@@ -83,7 +83,7 @@ Deprecations
 ~~~~~~~~~~~~
 
 - :meth:`DataFrame.to_stata`, :meth:`read_stata`, :class:`StataReader` and :class:`StataWriter` have deprecated the ``encoding`` argument.  The encoding of a Stata dta file is determined by the file type and cannot be changed (:issue:`21244`).
--
+- :meth:`Series.ptp` is deprecated. Use numpy.ptp instead (:issue:`21614`)
```

Comment:
you can put a reference to ``numpy.ptp`` as well

Reply:
done - updated for all comments Thanks